store/storage/rocksdb/opts.go

```diff
@@ -0,0 +1,122 @@
+package rocksdb
+
+import (
+	"encoding/binary"
+	"runtime"
+
+	"github.com/linxGnu/grocksdb"
+)
+
+const (
+	// CFNameStateStorage defines the RocksDB column family name for versioned state
+	// storage.
+	CFNameStateStorage = "state_storage"
+
+	// CFNameDefault defines the RocksDB column family name for the default column.
+	CFNameDefault = "default"
+)
+
+// NewRocksDBOpts returns the options used for the RocksDB column family for use
+// in state storage.
+//
+// FIXME: We do not enable dict compression for SSTFileWriter, because otherwise
+// the file writer won't report correct file size.
+// Ref: https://github.com/facebook/rocksdb/issues/11146
+func NewRocksDBOpts(sstFileWriter bool) *grocksdb.Options {
```

Comment:
Is there anything to say about how these many parameters were picked? Seems like something we should benchmark and tweak.

Reply:
https://github.com/crypto-org-chain/cronos/tree/main/versiondb/tsrocksdb